.eslintrc.json

```diff
@@ -31,6 +31,7 @@
 				"@typescript-eslint/no-extra-semi": "error",
 				"@typescript-eslint/no-unused-vars": "error",
 				"@typescript-eslint/no-non-null-assertion": "error",
+				"@typescript-eslint/no-explicit-any": "error",  // Added rule
```

Comment:
Please remove the `// Added rule` comment here too

Reply:
@rolivencia, I have made a PR , please checkout #918

Reply:
Why not adding the same changes to the existing #905 PR? Plus, the new #918 introduces unwanted changes in unrelated files.